dali/pipeline/executor/executor.cc

```diff
@@ -243,6 +243,44 @@ void Executor<WorkspacePolicy, QueuePolicy>::RunGPU() {
   QueuePolicy::QueueOutputIdxs(gpu_idxs, gpu_op_stream_);
 }
 
+template <typename WorkspacePolicy, typename QueuePolicy>
+void Executor<WorkspacePolicy, QueuePolicy>::RunCPU() {
```

Comment:
This is a bit repetitive; I tried some wrapper with a callable objects, but stack traces looked horrific.